lisa/features/gpu.py

```diff
@@ -275,6 +291,14 @@ def _install_cuda_driver(self, version: str) -> None:
                     self._node.os.install_packages("cuda-drivers-465")
                 else:
                     self._node.os.install_packages("cuda-drivers-515")
+        elif isinstance(self._node.os, CBLMariner):
+            # Replace "5.15.131.1-2.cm2" with "5.15.131.1.2.cm2"
+            kernel_ver = self._node.os.get_kernel_information().raw_version.replace(
+                "-", "."
+            )
+            self._node.os._install_package_from_url(
+                f"https://packages.microsoft.com/cbl-mariner/2.0/prod/nvidia/x86_64/Packages/c/cuda-525.85.12-3_{kernel_ver}.x86_64.rpm"
```

Comment:
does it support arm64? if yes, replace x86_64 into actual cpu type.

Reply:
No, it only supports x86. I will add that condition as well.